Kconfig.zephyr

```diff
@@ -338,6 +338,71 @@ endmenu
 
 menu "Compiler Options"
 
+config STD_AT_LEAST_C99
+	bool
+	help
+	  Hidden option to select compiler support C99 standard or higher.
```

Comment:
Would a better wording / name be `MINIMUM_REQUIRED_STD_C99` ?

Initially when reading `AT_LEAST` I got to think of something which the compiler / toolchain could be setting.